controllers/datadogagent/merger/fake/VolumeManager.go

```diff
@@ -0,0 +1,94 @@
+package fake
+
+import (
+	"testing"
+
+	commonv1 "github.com/DataDog/datadog-operator/apis/datadoghq/common/v1"
+	merger "github.com/DataDog/datadog-operator/controllers/datadogagent/merger"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+// VolumeManager is an autogenerated mock type for the VolumeManager type
+type VolumeManager struct {
+	VolumeByC      map[commonv1.AgentContainerName][]*v1.Volume
```

Comment:
curious if this needs to be by container?

Reply:
indeed we can remove by container for volume